common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -67,7 +67,11 @@
         BROKER_API_HELLO,
         BROKER_API_GET_BROKER_ACCOUNTS,
         BROKER_API_REMOVE_BROKER_ACCOUNT,
-        BROKER_API_UPDATE_BRT
+        BROKER_API_UPDATE_BRT,
+        BROKER_SET_FLIGHTS,
+        BROKER_GET_FLIGHTS,
+        BROKER_ADD_FLIGHTS,
+        BROKER_SSO_TOKEN;
```

Comment:
Here, I don't know what to call this.  It could be MSAL, but we're going to protect it, and only expect to use it for Edge.

Reply:
I'm going to make it MSAL_SSO_TOKEN, since we're exposing to to MSAL-Cpp.  I'm not sure that this API is cut in quite the right place.